middleware/assignLabels.js

```diff
@@ -0,0 +1,23 @@
+function setup(labelGenerator) {
```

Comment:
Yeah :+1: to having the default set within this module. Proxyquire seems to work pretty nice for testing, and keeps the interface much cleaner